src/fides/api/alembic/migrations/versions/f396c1f84b0f_add_banner_description_to_experience_.py

```diff
@@ -0,0 +1,41 @@
+"""Add banner_description to experience config
+
+Revision ID: f396c1f84b0f
+Revises: 4ced99dabebb
+Create Date: 2024-01-02 01:35:41.813279
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "f396c1f84b0f"
+down_revision = "4ced99dabebb"
```

Comment:
I think this will conflict with other open PRs so this is dangerous

Reply:
yep always good to flag thanks!

Reply:
Reconciled with open PR https://github.com/ethyca/fides/pull/4481